api.py

```diff
@@ -0,0 +1,32 @@
+import json
+from django.contrib.auth.decorators import user_passes_test
+from django.core import serializers
+from django.db import transaction, connection
+from django.http import HttpResponse
+from django.views.decorators.cache import never_cache
+from django.views.decorators.csrf import csrf_protect
+
+import commands
+
+@csrf_protect
+@never_cache
+@transaction.atomic
+@user_passes_test(lambda u: u.is_staff)
+def command(request):
+	data = json.loads(request.POST.get('data', '{}'))

```

Comment:
Maybe this might be a good time to bring up indentation standards. Personally, I'm a fan of the universal 2-space indent, but I see the new files here are tab-indent, and I'm not opposed to that either. I guess just a standard would be nice, but I'm not vindictive enough to specify one.